examples/yarn2/shortcuts.yarn

```diff
@@ -4,7 +4,7 @@ This is a test of shortcut functionality.
 
 -> Option 1
     Option 1 selected.
--> Option 2
+-> Option 2 <<if 0 > 1>>
     Option 2 selected.
 -> Option 3
```

Comment:
This PR looks great! Do you mind adding a test case with a conditional option that resolves to true?

Reply:
FYI, I fixed some issues with failing tests, so you may want to pull before making further changes :)